src/system/global_defs.h

```diff
@@ -152,50 +152,71 @@ if ( !(expr) ) \
 
 /// Definition of assert for debug mode only
 #ifdef FLOW123D_DEBUG_ASSERTS
-#define FEAL_DEBUG_ASSERT( expr) \
+#define FEAL_ASSERT_DBG( expr) \
 if ( !(expr) ) \
   feal::Assert( #expr).set_context( __FILE__, __func__, __LINE__)._FEAL_ASSERT_A
 #else
-#define FEAL_DEBUG_ASSERT( expr)
+#define FEAL_ASSERT_DBG( expr) \
+if ( !(expr) ) \
+  feal::AssertNull()._FEAL_ASSERT_A
 #endif
 
+/// Definition of comparative assert macro (Less Than)
+#define ASSERT_LT(a, b) \
```

Comment:
How to deal with conflict with gtest? For every macro without FEAL_ prefix we should check if it's not already defined, 
like that:
	#ifdefined ASSERT_LT
	#ifdefined FEAL_OVERRIDE_ASSERTS
		#undef ASSERT_LT
	#else 
		#warning "ASSERT_LT already defined out of FEAL."
        #endif
        #endif

These checks could possibly be done through another macro.
CHECK_DEFINITION(ASSERT_LT, FEAL_OVERRIDE_ASSERTS)